### ISSUE TITLE
[0.8.3] Fertilzers skill has no description

### ISSUE THREAD

**Issue by FireByTrial** [StrangeLoopGames/EcoIssues#14401]
### Description
When using the tooltip on Fertilizers Specialists the displayed information is 'None.' instead of the information I would expect to see like what is presented on other specializations.

### Steps to reproduce
Have Fertilizers researched

### Example of issue
![image](https://user-images.githubusercontent.com/698067/72448862-00a90680-3775-11ea-80f3-727da2a541cb.png)

**Comment by SlayksWood** [comment]
Thank you for report.
Fertilzers skill has description. In your case None means that nobody took specialty:
![image](https://user-images.githubusercontent.com/45708377/72496829-d1a99800-383b-11ea-82f4-a9c5fbb7d2b9.png)
It was already fixed in 0.9.0.

**Comment by FireByTrial** [comment]
@SlayksWood 🤞 its resolved in 0.9.0. I'll give more details on how to reproduce that I just discovered, the implementation is inconsistent it seems and corrects itself after the first occurrence. For me to reproduce this, I load into the game and mouse over a skill with nobody specializing in it (just me in my game), in that case 'None.' will display, if I mouse off of it and allow it to clear then mouse over it again, the description will not contain full text. I have attached two screenshots right after one another. As you mentioned this might already be resolved though. 

![20200116080233_1](https://user-images.githubusercontent.com/698067/72536580-be94c900-3837-11ea-8b65-9b77b2107b2e.jpg)
![20200116080236_1](https://user-images.githubusercontent.com/698067/72536593-c2285000-3837-11ea-878c-837cba5f148c.jpg)
